server.js

```diff
@@ -1,33 +1,107 @@
-import express from 'express'
-import bodyParser from 'body-parser'
-import cors from 'cors'
-
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
-const port = process.env.PORT || 8080
-const app = express()
+import express from 'express';
+import bodyParser from 'body-parser';
+import cors from 'cors';
+
+import booksData from './data/books.json';
+
+const port = process.env.PORT || 8080;
+const app = express();
 
 // Add middlewares to enable cors and json body parsing
-app.use(cors())
-app.use(bodyParser.json())
+app.use(cors());
+app.use(bodyParser.json());
 
-// Start defining your routes here
+// Routes start here:
+// Home: path /
 app.get('/', (req, res) => {
-  res.send('Hello world')
-})
+  res.send("Hello world, welcome to Vanessa's Bookish API! - See this API live at: https://vane-bookish-app.netlify.app/")
+});
```

Comment:
It would be icing on the cake if you'd include documentation! This lib is pretty cool -> https://www.npmjs.com/package/express-list-endpoints